package.json

```
@@ -12,6 +12,7 @@
     "dev:server": "nx run server:dev",
     "lint": "nx run-many --target=lint --all",
     "start": "node dist/packages/server/main.js",
+    "test": "NODE_ENV=development NX_ENABLE_MOCKS=1 nx run-many --target=dev --all",
```

You could rename it like this, since it doesn't really start automated test. 
```suggestion
    "dev:mock": "NODE_ENV=development NX_ENABLE_MOCKS=1 nx run-many --target=dev --all",
```